src/client/App.js

```diff
@@ -18,6 +18,7 @@ class App extends Component {
         <Header />
         <Navmenu />
         <SearchForm />
+        <Calendar />
```

Comment:
Calendar should not go here.  The component should appear when we click the calendar view button. We will redo the events view later